Source/DafnyLanguageServer/Workspace/ICompilationStatusNotificationPublisher.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.Dafny.LanguageServer.Workspace.Notifications;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+namespace Microsoft.Dafny.LanguageServer.Workspace {
+  /// <summary>
+  /// Implementations of this interface are responsible to publish the compilation
+  /// status of a <see cref="DafnyDocument"/> to the LSP client.
+  /// </summary>
+  public interface ICompilationStatusNotificationPublisher {
```

Comment:
Do you need this interface?

Reply:
Yes, as this interface and its implementation is used by [TextDocumentLoader.cs](https://github.com/dafny-lang/dafny/blob/4a0d3d82ba6fb69231e7779e5d599fe5122ad595/Source/DafnyLanguageServer/Workspace/TextDocumentLoader.cs#L23).